File: src/Unidesk/Unidesk/Dtos/UserInTeamDto.cs
﻿using System.ComponentModel.DataAnnotations;
using Unidesk.Db.Models;
using Unidesk.Dtos.ReadOnly;
using Unidesk.Server;

namespace Unidesk.Dtos;

public class UserInTeamDto : UserLookupDto
{
    public UserInTeamStatus Status { get; set; }
    public TeamRole Role { get; set; }
}

File: src/Unidesk/Unidesk.Client/src/api-client/models/UserInTeamDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { TeamRole } from './TeamRole';
import type { TeamSimpleDto } from './TeamSimpleDto';
import type { UserDto } from './UserDto';
import type { UserInTeamStatus } from './UserInTeamStatus';
import type { UserSimpleDto } from './UserSimpleDto';

export type UserInTeamDto = {
    readonly _DtoType?: string | null;
    userId?: string;
    user?: (UserSimpleDto | UserDto) | null;
    teamId?: string;
    team?: TeamSimpleDto;
    status?: UserInTeamStatus;
    role?: TeamRole;
    readonly id: string;
};
